incubator/spark-history-server/charts/nfs/templates/nfs-server-rc.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.enableExampleNFS }}
+apiVersion: v1
+kind: ReplicationController
```

Comment:
Instead of a `ReplicationController` can you please use a `Deployment`. This is the recommended controller to use now. See the docs at https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller/.

While `ReplicationController` is still present for API compatibility it is generally considered something that should not be used.